pkgs/development/ocaml-modules/faraday/default.nix

```diff
@@ -2,22 +2,18 @@
 
 buildDunePackage rec {
   pname = "faraday";
-  version = "0.8.1";
-
-  useDune2 = true;
-
-  minimumOCamlVersion = "4.02";
```

Comment:
Why are you removing this ? does it build with 4.0 ?

Reply:
What do you mean by “better”?

Reply:
```
$  nix build github:vbgl/nixpkgs/ocaml-faraday-0.8.2#ocaml-ng.ocamlPackages_4_07.faraday
error: fmt is not available for OCaml 4.07.1
```
if we set this to `4.08.0` then we get `error: faraday-0.8.2 is not available for OCaml 4.07.1`

Reply:
That’s right. But you don’t explain why you think one of those is better than the other.

Reply:
In the first case as users that want package `faraday` I receive error message that cite `fmt` from user point of view it is unrelated unless it dig and see it is a dependencies. I find more explicit but you can ignore if you are not convinced.

Reply:
Fixed. Thanks.